CCScriptWriter/CCScriptWriter.py

```diff
@@ -616,123 +560,30 @@ def outputToCoilSnakeProject(self):
                                 f = self.dataFiles[b]
```

Comment:
As done on other branches of CCScriptWriter
```suggestion
                                f = None
                                try:
                                    f = self.dataFiles[b]
                                except KeyError:
                                    raise ValueError("While processing {fileName} sector {e}.{s} door {n}, couldn't find a data CCS file containing 'l_{hex(b)}'. Is this a valid label?")
```
and similarly `"While processing {fileName} entry {e}, couldn't a data CCS file containing 'l_{hex(v)}'. Is this a valid label?"` for the instance above it. GitHub won't let me insert a suggestion up there.

Since you have implemented this suggestion on other branches, though, you might want to keep it there until those branches are merged?

Reply:
Implemented it here - I'll probably not rebase `charmap` on top of this since it is destined to be replaced by my `jp-support` branch sometime soon. I'll rebase `jp-support` on the latest changes here.